src/utils/endpoints.js

```diff
@@ -0,0 +1,184 @@
+export default function endpoints() {
+  const collaborator = '/api/collaborator';
+  const product = '/api/product';
+  const category = '/api/category';
+  const lot = '/api/lot';
+  const belongs = '/api/belongs';
+
+  const endpoints = [
+    {
+      collaborator: [
+        {
+          type: 'POST',
+          endpoint: `${collaborator}`,
+          parameters: [
+            {
+              idAcessLevel: 'Integer',
+              idCompany: 'Integer',
+              name: 'String',
+              document: 'String',
+              password: 'String',
+            },
+          ],
+          description: 'Creates new collaborator.',
+        },
+        {
+          type: 'POST',
+          endpoint: `${collaborator}/auth`,
+          parameters: [
+            {
+              document: 'String',
+              password: 'String',
+            },
+          ],
+          description: 'Verifies user authentication.',
+        },
+        {
+          type: 'GET',
+          endpoint: `${collaborator}s/:idCompany`,
+          parameters: [
+            {
+              idCompany: 'Integer',
+            },
+          ],
+          description:
+            'Returns a list of all collaborators for a specific Company.',
```

Comment:
Aqui seria   "description": "Returns a list of all collaborators of a specific Company.", não?